common/source/docs/common-brushless-escs.rst

```diff
@@ -1,121 +1,94 @@
 .. _common-brushless-escs:
 
-====================================
-PWM,OneSHot, and DShot Protocol ESCs
-====================================
+================================
+PWM, OneShot and OneShot125 ESCs
+================================
 
-The most common electric motor used with ArduPilot vehicles is brushless and requires a brushless ESC for control. The main protocols for controlling these ESCs are PWM, OneShot, and DShot. In addition, some include the ability to be configured via an external PC application called BLHeli Suite or BLHeli32 Suite. See :ref:`common-blheli32-passthru` for more information on ESCs with that capability, but the following sections apply to ArduPilot's setup for those ESCs also.
+Most ArduPilot vehicles use brushless motors controlled by brushless motor ESCs. The most common protocols used by these ESCs are PWM, OneShot, OneShot125, :ref:`DShot <common-dshot-escs>` and :ref:`BLHeli <common-blheli32-passthru>`. This page describes the first three (PWM, OneShot and OneShot125).
```

Comment:
mixing apples and oranges...BLHeli is not an ESC control protocol.....
```suggestion
Most ArduPilot vehicles use brushless motors controlled by brushless motor ESCs. The most common protocols used by these ESCs are PWM, OneShot, OneShot125,  and :ref:`DShot <common-dshot-escs>`. This page describes the first three (PWM, OneShot and OneShot125).
```

Reply:
While I mostly agree I'm also not sure what better wording to use.  I could change **protocols used by these** to **types** so it becomes, "Most ArduPilot vehicles use brushless motors controlled by brushless motor ESCs. The most common types of ESCs are PWM, OneShot, OneShot125, and DShot and BLHeli"

I think we don't need to be perfect as long as users do the right thing.  "DShot" certainly is a protocol.. BLHeli seems to be many things.. multiple projects, ESC software, a configuration GUI, the pass-through seems similar to a protocol to me...

I don't know.. it's tricky and I think I've only recently understood the difference as part of rewriting these wiki page.

Reply:
DShot is the protocol. DShot with RPM feedback is also a protocol. BLHeli is not

Reply:
ok, I've removed mention of BLHeli from the "PWM, OneShot, OneShot125" page.